qml/keys/SpaceKey.qml

```diff
@@ -56,6 +56,8 @@ ActionKey {
         }
 
         onPressed: {
+            if (maliit_input_method.useHapticFeedback)
```

Comment:
Please use open and closing braces around the block in if statements, to keep consistent with the rest of the file, like so:
```
if (foo) {
    do_something();
}
```

Reply:
Also, how does this behave when doing long press? The vibration keeps happening until the key is released?

Reply:
I will clean that up when I get a chance this weekend. Thabks for the pointers.

As for behaviour, no, it vibrates just once when pressed and works as expected in touchpad mode. I really just copy+pasted the line from another key's file, as the lack of feedback was annoying me; it shouldn't cause any trouble.